src/azure-cli/azure/cli/command_modules/acr/_utils.py

```diff
@@ -263,11 +263,11 @@ def get_yaml_template(cmd_value, timeout, file):
                 yaml_template += s
         else:
             if os.path.exists(file):
-                f = open(file, 'r')
-                for line in f:
-                    yaml_template += line
+                with open(file, 'r') as f:
+                    for line in f.readlines():
```

Comment:
`for line in f:` is preferred. `f.readlines()` is less efficient: https://docs.python.org/3/tutorial/inputoutput.html#methods-of-file-objects